### DISCUSSION TITLE
About page

### DISCUSSION THREAD

**Discussion by johannadm** [netwerk-digitaal-erfgoed/solid-crs#375] [Presentation]
If there would an about page, with more details and info about the heritage institution, where would the link best be placed? And what specific content is necessary to include? 

Example for where the link could be placed: underneath the name of the institution
![afbeelding](https://user-images.githubusercontent.com/80322390/126645122-4f73fa0b-2158-4689-aa1a-5db2fa08ed39.png)

Example of the about page:
![afbeelding](https://user-images.githubusercontent.com/80322390/126645187-dbe58681-389a-4b20-8994-10c49894997f.png)

**Comment by ddeboer** [comment]
@johannadm I like the link in the sidebar, but ‘Over de vereniging’ may not be generic enough. Perhaps ‘Over de instelling’ or simply ‘Over’?

IIRC, we also discussed an about snippet on the landing page (the topmost screen). It could either be (slightly confusing, perhaps) an extra tile like the collection ones or be placed in the grey background container. The snippet should show the institute’s title (which is already in the sidebar, but first-time visitors may overlook that) and its `schema:description` (abbreviated if necessary). The whole snippet can then link to the About page, too.

**Reply by johannadm** [reply]
![afbeelding](https://user-images.githubusercontent.com/80322390/127174891-adca3b24-c9bb-41aa-bc3b-e3da83564c72.png)
Does this mock-up show what you had in mind? It seems better with the link to more information in the text instead of in the primary navigation. What do you think? :)

**Reply by ddeboer** [reply]
Yes! Some points:

- I prefer to keep the About link in the primary navigation though, particularly for users that land on a deep link (for example, from Google).
- The page’s header should still be the old one, so ‘Collecties: overzicht van alle collecties’.
- Can you show the organization’s logo in the header and on the about page (https://github.com/netwerk-digitaal-erfgoed/solid-crs/discussions/397)? If no organization logo is available, we can fall back to the NDE logo.

**Reply by lem-onade** [reply]
Hey @ddeboer 

This is the current state of the 'about' page, which makes use of the extended data model for orgs:

![image](https://user-images.githubusercontent.com/32491056/128992927-18a51f1d-8aaf-43bb-9b66-cdf55fb0debf.png)

The logo of the institution can be found in the header as well as the name. Description, website, email and telephone can be found in the top card of the about page.

I have left out the images of objects under the collections as that made the page less clean. (messy, really)

Is this what you had in mind? Do you feel anything should change?
